contrib/epee/include/net/jsonrpc_server_impl_base.h

```diff
@@ -24,10 +24,11 @@ template<class t_child_class, class t_connection_context = epee::net_utils::conn
         : m_net_server(external_io_service)
     {}
 
-    bool init(const std::string& bind_port = "0", const std::string& bind_ip = "0.0.0.0")
+    bool init(std::function<void(size_t, uint8_t*)> rng, const std::string& bind_port = "0", const std::string& bind_ip = "0.0.0.0")
```

Comment:
Is this one necessary? HTTP auth is never used by this server, this is JSON-RPC directly over TCP. In fact, this shouldn't compile at all, but shouldn't matter since this is currently unused.